manifests/vhost.pp

```diff
@@ -539,11 +539,19 @@
     }
   }
 
+  # Check if mod_env is required and not yet loaded.
+  # create an expression to simplify the conditional check
+  $use_env_mod = $setenv and ! empty($setenv)
+  if ($use_env_mod) {
+    if ! defined(Class['apache::mod::env']) {
+      include ::apache::mod::env
+    }
+  }
   # Check if mod_setenvif is required and not yet loaded.
   # create an expression to simplify the conditional check
-  $use_setenv_mod = ($setenv and ! empty($setenv)) or ($setenvif and ! empty($setenvif)) or ($setenvifnocase and ! empty($setenvifnocase))
+  $use_setenvif_mod = ($setenvif and ! empty($setenvif)) or ($setenvifnocase and ! empty($setenvifnocase))
```

Comment:
@sathieu  This appears to be a change in behavior. Previously, the `use_setenv_if_mod` would only be `true` if `use_setenv_mod` were true. Now it is evaluated independently and below... can add the config fragment without the setting of `$setenv`.

Is this desired behavior?